app/views/shared/forms/_savings_and_investments_form.html.erb

```diff
@@ -1,26 +1,34 @@
 <%= render partial: 'accounts_summary', locals: { bank_accounts: bank_accounts } if bank_accounts.any? %>
 
-<%= form_with(model: @form,
-              url: form_path,
-              method: :patch,
-              local: true) do |form| %>
+<%= form_with(model: @form, url: form_path, method: :patch, local: true) do |form| %>
 
-  <%= govuk_form_group do %>
-    <div class="govuk-checkboxes" data-module="checkboxes">
-      <%= render(
-            partial: 'shared/forms/revealing_checkbox/attribute',
-            collection: attributes,
-            locals: {
-              model: @form,
-              form: form
-            }
-          ) %>
-    </div>
+  <%= govuk_form_group show_error_if: (@form.errors.present? && !@form.any_checkbox_checked?) do %>
+    <%= govuk_fieldset_header do %>
+      <h1 class="govuk-fieldset__heading govuk-!-padding-bottom-7"><%= page_title %></h1>
+    <% end %>
+    <fieldset class="govuk-fieldset" aria-describedby="citizenship-conditional-hint">
```

Comment:
I can't find an id of "citizenship-conditional-hint" anywhere in this PR, and cannot find "citizenship" anywhere in the code base. So I cannot see how this `aria-describedby` can be correct.